packages/core/src/site/graphql-robots-service.test.ts

```diff
@@ -65,5 +68,23 @@ describe('GraphQLRobotsService', () => {
 
       return expect(nock.isDone()).to.be.true;
     });
+
+    it('should pass fetchOptions to the GraphQL client request', async () => {
+      const fetchOptions = {
```

Comment:
We need to verify that `fetchOptions` are passed to `graphqlClient.request`
currently we check that it's passed to the method, but we are not verifying if it's applied